runtime/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
```

Comment:
This file is very useful to debug multi-threaded workflows. Surprisingly the runtime package was missing this file. It is only used in the maven testing phase so it doesn't affect the maven compile and package goals.

This does not cause any test logs overhead as it has all the appenders commented out at the bottom of the file.
As you know, we usually enable the appenders only when we need them.

Reply:
I don't think there is anything peculiar about debugging multi threaded code. We simply want to have logging for the testing module, we used to have it, but it seems like at some point it was removed.

Can you move the dependency to the test module instead of adding it to the runtime and changing the stage scope.

Also, wondering if this will break some metrics related functionality. Not clear if this dependency was removed for a specific reason https://github.com/CorfuDB/CorfuDB/pull/2841/files#diff-63dc8be7e50c9d6685f98818cd088168fd9b2ead857e9e7dbe5f7d4fd682d16dL67

Reply:
1. For multi-threaded code, we cannot have breakpoints while debugging, we rely on logs to check what's happening with a background thread.

2. test module already has in `test/pom.xml-
```
            <groupId>ch.qos.logback</groupId>
            <artifactId>logback-classic</artifactId>
            <version>${logback.classic.version}</version>
            <scope>test</scope>
```
        
I needed it for the tests in the runtime module, hence added it in runtime/pom.xml.

3. Checking with @PavelZaytsev to see if it breaks anything.

Reply:
Ok I think I added logback classic at some point to the runtime pom to make it work easier with loggers when it came to metrics. But when building corfu runtime as nsx dep it introduced some build conflicts because they use some other library instead, I think slf4j. If you want to have it part of test module I think its fine but you will have bazel build issue when you try to create a new artifact in gerrit